### PR TITLE
[BLOG] Add moving-scipy-to-meson.md

### PR DESCRIPTION
**Pull request by MarsBarLee**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

## Links to other Quansight blogs:

no labs links found
no llc links found

### CODE REVIEW

**Comment by MarsBarLee** [comment]
@gabalafou This is ready for review. Also, could you give me permissions to add labels to issues, if possible? As of now, I cannot label issues with the relevant 'type: content 📝'

**Comment by bskinn** [comment]
@gabalafou  This also resurfaces the need to make whatever administrative changes are necessary to allow a branch-and-merge strategy for Labs blog contributions, instead of fork-and-merge, to avoid colliding with Vercel's fork protection.

**Comment by MarsBarLee** [comment]
@gabalafou Yes, I can add labels now, thank you!

File: apps/labs/posts/moving-scipy-to-meson.md [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+title: 'Moving SciPy to the Meson build system'
+published: July 25, 2021
+author: ralf-gommers
+description: 'SciPy now builds with Meson on Linux, and the full test suite passes!
+This is a pretty exciting milestone, and good news for SciPy maintainers and
+contributors - they can look forward to much faster builds and a more
+pleasant development experience.'
+category: [Developer workflows]
+featuredImage:
+  src: ../public/posts/moving-scipy-to-meson/feature.png
```
Comment (gabalafou) [inline apps/labs/posts/moving-scipy-to-meson.md:11]:
```suggestion
  src: /posts/moving-scipy-to-meson/feature.png
```

File: apps/labs/posts/moving-scipy-to-meson.md [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+  alt: ''Cropped diagram of SciPy's external build and runtime dependencies.'
+hero:
+  imageSrc: ../public/posts/moving-scipy-to-meson/blog_hero_var2.svg
```
Comment (gabalafou) [inline apps/labs/posts/moving-scipy-to-meson.md:14]:
```suggestion
  imageSrc: /posts/moving-scipy-to-meson/blog_hero_var2.svg
```

File: apps/labs/posts/moving-scipy-to-meson.md [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+published: July 25, 2021
+author: ralf-gommers
+description: 'SciPy now builds with Meson on Linux, and the full test suite passes!
+This is a pretty exciting milestone, and good news for SciPy maintainers and
+contributors - they can look forward to much faster builds and a more
+pleasant development experience.'
+category: [Developer workflows]
+featuredImage:
+  src: ../public/posts/moving-scipy-to-meson/feature.png
+  alt: ''Cropped diagram of SciPy's external build and runtime dependencies.'
```
Comment (gabalafou) [inline apps/labs/posts/moving-scipy-to-meson.md:12]:
```suggestion
  alt: 'Cropped diagram of SciPy's external build and runtime dependencies.'
```

File: apps/labs/posts/moving-scipy-to-meson.md [hunk 4]
```diff
@@ -0,0 +1,497 @@
...
+one of the goals of this exercise is to get away from distutils/setuptools
+completely and use only a modern build system and Python packaging standards
+(i.e. `pyproject.toml`-based builds) to interact with `pip` and other Python
+packaging tools.
+
+So in February I wrote
+[an RFC titled "switch to Meson as a build system"](https://github.com/scipy/scipy/issues/13615),
+and after positive feedback started working on this project.
+
+<img src="../public/posts/moving-scipy-to-meson/scipy_build_dependencies.png" alt="Diagram of SciPy's build and runtime dependencies" width="600">
```
Comment (gabalafou) [inline apps/labs/posts/moving-scipy-to-meson.md:88]:
```suggestion
<img src="../public/posts/moving-scipy-to-meson/scipy_build_dependencies.png" alt="Diagram of SciPy's build and runtime dependencies" width="600" />
```

Comment (gabalafou) [reply apps/labs/posts/moving-scipy-to-meson.md:88]: @noatamir here's an example of an unclosed `` tag that our markdown compiler doesn't like.

**Pull request closed without merging**